### DISCUSSION TITLE
网络异常的问题

### DISCUSSION THREAD

**Discussion by JacianZhu** [simple-robot/Discussions#153] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

本机可以正常扫码登录，换到服务器上就提示网络异常
![IMG_4903](https://github.com/simple-robot/Discussions/assets/51085457/bae3c20f-b4fd-4c9e-b3a0-345ea0908c09)

**Comment by ForteScarlet** [comment]
字面意思，手表扫码可能会要求在同一局域网
